deploy/releases/daily/openliberty-app-operator.yaml

```diff
@@ -55,18 +53,6 @@ rules:
   - deployments/finalizers
   verbs:
   - update
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - get
-- apiGroups:
-  - apps
-  resources:
-  - replicasets
-  verbs:
-  - get
```

Comment:
@arturdzm @navidsh This portion was removed from role and cluster role. Just want to confirm that it's not needed.

Reply:
I don't think we do anything specific to `replicasets`. Not sure about `pod`. Isn't that needed for trace/dump?